Dockerfile

```diff
@@ -41,6 +53,7 @@ ADD openvpn/ /etc/openvpn/
 ADD transmission/ /etc/transmission/
 ADD tinyproxy /opt/tinyproxy/
```

Comment:
@haugene should we still keep tinyproxy as well?

Reply:
@pkishino Keeping TinyProxy or not will come down to how much the HTTP auth feature means to you/ other users (`WEBPROXY_USERNAME` and `WEBPROXY_PASSWORD`) This can be supported by setting Privoxy to go upstream to TinyProxy, in which case we would need both services. Otherwise Privoxy can run fine without TinyProxy, which is how I run my instance (and tunnels through the VPN just fine in the same way TinyProxy does).

This is because Privoxy does not contain built in support for HTTP proxy auth. (see https://www.privoxy.org/user-manual/config.html 7.4.9) and you would need to use the `enable-proxy-authentication-forwarding` feature of Privoxy as a part of sending the HTTP auth. provision upstream (to TinyProxy). Personally I'm fine without HTTP auth. which isn't that secure anyway and Privoxy instead supports acccess control by IP address via `permit-access` and `deny-access` directives, if you wanted to do that at the Privoxy level.

As for how to config, it is all done in text files so things can continue in a similair fashion to the present `sed`-ing methods, but I am not an expert at using `sed` so would appreciate some help there :)